package.json

```diff
@@ -20,15 +18,16 @@
   },
   "packageManager": "pnpm@8.15.5",
   "devDependencies": {
+    "@biomejs/biome": "1.8.0",
     "husky": "^9.0.7",
-    "lint-staged": "^15.2.0",
-    "prettier": "^3.2.4"
+    "lint-staged": "^15.2.2"
   },
   "scripts": {
     "prepare": "husky",
-    "lint-staged": "lint-staged --relative",
-    "lint": "prettier . --check",
-    "format": "prettier . --write"
+    "lint": "biome check .",
+    "fix": "biome check --write",
+    "format": "biome format --write",
```

Comment:
Should these have `.` passed, like `lint` does? (Or should `lint` not pass `.`?)